LFS171x/indy-material/nodejs/AgentDemoScript.md

```diff
@@ -0,0 +1,319 @@
+
+
+### Step 3: Applying for the Job
+
+Alice has gotten her transcript from Faber. Now all she has to do is land the job! She has interviewed successfully (or so she thinks) and it all comes down to her transcript. No pressure. Just as she did with Faber, Alice must first establish a relationship with Acme.
+
+
+
+
+1.  Click **Send Connection Request**. 
+
+|![CreateRelBlank](images/CreateRelBlank.png "Blank Create Relationship Popup")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+1.  Copy and paste Acme's Endpoint DID and click **Send Connection Request**.
+
+|![CreateRelFill2](images/CreateRelFill2.png "Acme Create Relationship Popup")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+Connection established! As with the Alice-Faber connection, immediately after the connection is established, they both send a Proof Request to the other to get information from the government credential they each hold.
+
+Alice checks her messages (refresh browser tab) and sees that she has a Proof Request from Acme for the "Name" attribute. 
+
+|![NamePR](images/NamePR.png "Accept Name Proof Request Popup")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+
+
+1.  Click **Messages** and then **Accept**.
+
+Acme does the same - checks messages, sees the Proof Request and clicks Accept.  Relationship established!
+
+Acme then sends a Proof Request to get Alice's transcript. Here we have to use a bit of a hack for this demo to get the scenario to work.  This is NOT how this is going to work in a real app, but this is enough to get the idea across.
+
+So far in this demo, the Proof Requests have "just happened."  That is, when a connection is made, both parties send a Proof Request to each other. In this case, we're going to walk through the steps of Acme requesting the proof.  What's the problem with the demo?  Unfortunately, in this demo, Acme doesn't know the information to construct the Proof Request for the transcripts, so they have to get that information from Faber. Worse, we're going to use copy and paste to get that information.  That's ugly! Rest assured, there are other ways for this to work in a real app, but for now, we'll just use this approach.
+
+Start this step in the Faber browser tab.  Click on **Proof Requests**. In the **Select a Proof Request** dropdown, select "Transcript-Data." The text (a JSON structure) in the box to the right will update to begin with "name": "Transcript-Date". Highlight all of the data in that text box and copy the text (right-click and choose "copy").  Next, go over to Acme to request the Transcripts Proof.
+
+From the Acme browser tab:
+
+
+
+1.  Click **Proof Requests**
+
+|![PastePR](images/PastePR.png "Paste PR from Faber to ACME")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+1.  Select the **Relationship**, in this case, **Alice**.

```

Comment:
Want to continue the previous list. Indent above image.

Reply:
Fixed